recipes/azureml/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
```

Comment:
Please regenerate this recipe with grayskull:

```
grayskull pypi --strict-conda-forge azureml
```

Reply:
done